commands/generate.js

```diff
@@ -0,0 +1,101 @@
+'use babel';
+
+const fs = require('fs');
+
+var getProjectRoot = function () {
+	return atom.project.getPaths()[0];
+};
+
+var getConfig = function () {
+	return new Promise(function(resolve) {
```

Comment:
Why are you (ab)using promises for sync operations? All of these functions can just be normal functions returning a value. I don't see any async operations here.